src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String output;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName));
+            StringBuilder stringBuilder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
```

Comment:
```suggestion
              String[]  words = line.toLowerCase().split("\\W+");
              // for each loop and add word to builder if it starts correctly (user str.startWith())
```
this way we can shorten the code. You will only need to split words later one and sort them 
P.S. regex is better to declare as constant

Reply:
add